hack/ci-deploy-release.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+#
+# Copyright (C) 2024 ScyllaDB
+#
+# This script deploys scylla-operator stack.
+# Usage: ${0} <operator_image_ref>
+# (Avoid using rolling tags.)
+
+set -euxEo pipefail
+shopt -s inherit_errexit
+
+source "$( dirname "${BASH_SOURCE[0]}" )/lib/kube.sh"
+
+if [[ -n "${1+x}" ]]; then
+    operator_image_ref="${1}"
+else
+    echo "Missing operator image ref.\nUsage: ${0} <operator_image_ref>" >&2 >/dev/null
+    exit 1
+fi
+
+source_raw="$( skopeo inspect --format='{{ index .Labels "org.opencontainers.image.source" }}' "docker://${operator_image_ref}" )"
+if [[ -z "${source_raw}" ]]; then
+    echo "Image '${operator_image_ref}' is missing source label" >&2 >/dev/null
+    exit 1
+fi
+source_url="${source_raw/"://github.com/"/"://raw.githubusercontent.com/"}"
+
+revision="$( skopeo inspect --format='{{ index .Labels "org.opencontainers.image.revision" }}' "docker://${operator_image_ref}" )"
+if [[ -z "${revision}" ]]; then
+    echo "Image '${operator_image_ref}' is missing revision label" >&2 >/dev/null
+    exit 1
+fi
+
+ARTIFACTS="${ARTIFACTS:-$( mktemp -d )}"
+
+kubectl_create -n=prometheus-operator -f="${source_url}/${revision}/examples/third-party/prometheus-operator.yaml"
+kubectl_create -n=haproxy-ingress -f="${source_url}/${revision}/examples/third-party/haproxy-ingress.yaml"
+
+kubectl_create -f="${source_url}/${revision}/examples/common/cert-manager.yaml"
+# Wait for cert-manager crd and webhooks
+kubectl wait --for condition=established --timeout=60s crd/certificates.cert-manager.io crd/issuers.cert-manager.io
+for d in cert-manager{,-cainjector,-webhook}; do
+    kubectl -n=cert-manager rollout status --timeout=5m "deployment.apps/${d}"
+done
+wait-for-object-creation cert-manager secret/cert-manager-webhook-ca
+
+mkdir "${ARTIFACTS}/operator"
+cat > "${ARTIFACTS}/operator/kustomization.yaml" << EOF
+resources:
+- ${source_url}/${revision}/deploy/operator.yaml
+patches:
+- patch: |-
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: scylla-operator
+      namespace: scylla-operator
+    spec:
+      template:
+        spec:
+          containers:
+          - name: scylla-operator
+            image: "${operator_image_ref}"
+            env:
+            - name: SCYLLA_OPERATOR_IMAGE
+              value: "${operator_image_ref}"
```

Comment:
is this missing the tweaks from ci-deploy (https://github.com/scylladb/scylla-operator/blob/master/hack/ci-deploy.sh#L34-L35) on purpose? If so, is it superfluous there or accidentally missing here?